build/web_compose/Dockerfile

```diff
@@ -105,5 +118,5 @@ ENV DC_API_ROOT=https://api.datacommons.org
 ENV ENABLE_ADMIN=true
 
 COPY build/web_compose/compose.sh .
-EXPOSE 8080 8081
+EXPOSE 6060 8080 8081
```

Comment:
actually should only expose 8080 now?

mixer and nl server should be internal servers now

Reply:
Expose only documents that the container uses these ports. They are published via the `-p` command line argument.

Reply:
Oh, you are right.